pages/docs/configuration/configuration-acceleration.md

```diff
@@ -26,17 +26,25 @@ All data communicated within a coupling scheme needs to be configured through `e
+* For a **parallel coupling**, all coupling data is accelerated the same way. This means all coupling data use the same coefficients for the linear combination.
+* For a **serial coupling** only coupling that is exchanged from the `second` to the `first` participant is accelerated. Coupling data exchanged in the other direction (from `first` to `second`) is not modified.
 
-Let's look at an example: For fluid-structure interaction, if we first execute the fluid solver with given interface displacements followed by the structure solver taking forces from the fluid solver and computing new interface displacements, (only) the displacements are post-processed in case of serial coupling. For parallel coupling, both displacements and forces are post-processed.
+**Example:** For fluid-structure interaction, if we first execute the fluid solver with given interface displacements followed by the structure solver taking forces from the fluid solver and computing new interface displacements, (only) the displacements are accelerated in case of serial coupling. For parallel coupling, both displacements and forces are accelerated.
 
-Next, we have to configure based on which data the acceleration computes, i.e. how the coefficients in the linear combinations get computed. These data fields are defined within the acceleration as `data` tags (such as `Displacements` in the code example above). Let's call these data fields **primary data**. (Just for completeness: All coupling data that gets post-processed and that is not primary data, is called "secondary data".)
+Different acceleration schemes compute these coefficients in different ways, which can be grouped into two categories:
 
-* For **serial coupling**, you can only configure one primary data field, which should correspond to a coupling data field that is exchanged from the `second` to the `first` participant. In the FSI example, the `Displacements`.
-* For **parallel coupling**, an arbitrary number of primary data can be configured. For numerical performance reasons, you should define at least one coupling data field of each direction (one from `second` to `first`, one from `first` to `second`). In the FSI example, configure `Displacements` and `Forces`.
+* Coefficients are fixed for each iteration, thus independent of the accelerated values. Example being constant under-relaxation.
+* Coefficients depend on the values of previous iterations. Example being Interface Quasi-Newton methods.
+
+Data-dependent acceleration schemes need to select which data to compute these coefficients from by listing them as `data` tags inside the `acceleration` tag.
+We call data which influences the coefficients **primary data** and data which is accelerated without influencing the coefficients **secondary data**.
+In the code example above, `Displacements` is primary data and `Forces` is secondary data.
```

Comment:
The "why" would be important here.
How can the user affect what will be primary and which secondary?

Reply:
This is only true for serial coupling. The example above considers both serial and parallel coupling.

Reply:
The code example is in fact `serial-implicit`.

```suggestion
In the serial code example above, `Displacements` is primary data and `Forces` is secondary data.
```

This still confuses me though. The second participant receives Forces, does its step and then accelerates the data. Why would it accelerate `Forces`?

Reply:
Data sent from first to second in a serial scheme is never accelerated. Thus, it is neither primary nor secondary, but not related to the acceleration at all. That's still the confusing part, right?
(so the current statement is actually still wrong)

Reply:
Understood :+1: 

```suggestion
In the code example above, `Displacements` is primary data and there is no secondary data.
```

Reply:
```suggestion
In the code example above, for serial coupling, `Displacements` is primary data there is no secondary data. For parallel coupling, on the other hand, both data fields are primary data.
```

Maybe it makes more sense to use a new example here (simply give configuration of acceleration) with primary and secondary data?

Reply:
I had a look at our tutorials, and we don't seem to have one with a serial coupling scheme and secondary data.

I would propose to add another data exchanged from second to first in our example above.

Reply:
I added `Density` as some material property to the code example and moved the explanation to the bottom. IMHO the result flows better.

Reply:
Moving to other threads